docs/src/components/buttons/OpenSandboxButton.js

```diff
@@ -0,0 +1,30 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { IconButton, Tooltip } from 'gestalt';
+import trackButtonClick from './trackButtonClick.js';
+
+type Props = {|
+  onClick: () => void,
+|};
+
+export default function OpenSandboxButton({ onClick }: Props): Node {
+  const label = 'Open in CodeSandbox';
+
+  return (
+    <Tooltip inline text={label} idealDirection="up">
```

Comment:
This should be fixed to follow Tooltip best practices. They should not be repeated. We can omit `accessibilityLabel=""`

Reply:
out of scope